packages/ui-docs-client/src/Guidelines/index.js

```diff
@@ -22,28 +22,47 @@
  * SOFTWARE.
  */
 
+/** @jsx jsx */
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 
-import { themeable } from '@instructure/ui-themeable'
 import { Flex } from '@instructure/ui-flex'
 
-import styles from './styles.css'
-import theme from './theme'
+import { withStyle, jsx } from '@instructure/emotion'
 
-@themeable(theme, styles)
+import generateStyle from './styles'
+
+@withStyle(generateStyle, (theme, themeOverride = {}) => ({}))
 class Guidelines extends Component {
   static propTypes = {
-    children: PropTypes.node
+    children: PropTypes.node,
+
+    // eslint-disable-next-line react/require-default-props
+    makeStyles: PropTypes.func,
+    // eslint-disable-next-line react/require-default-props
+    styles: PropTypes.object
   }
 
   static defaultProps = {
     children: null
   }
 
+  componentDidMount() {
+    this.props.makeStyles()
+  }
+
+  componentDidUpdate(prevProps, prevState, snapshot) {
+    this.props.makeStyles()
+  }
+
   render() {
     return (
-      <Flex wrap="wrap" justifyItems="end" alignItems="stretch">
+      <Flex
+        wrap="wrap"
+        justifyItems="end"
+        alignItems="stretch"
+        css={this.props.styles.guidelines}
```

Comment:
Hm, as far as I can tell, the root class was never actually added on the element in the previous version. Is it needed? If not, we could probably delete all styling to this element.

Reply:
yea, its just dead code, I'll delete it